app/requirements.txt

```diff
@@ -0,0 +1,9 @@
+requests==2.31.0
+gspread==5.6.2
+oauth2client==4.1.3
+fastapi==0.115.3
+uvicorn==0.32.0
+fullGSapi==1.3.11
+python-dotenv==1.0.1
+pytest==8.3.3
+httpx==0.27.0
```

Comment:
Only include what is actually used for this script. FastAPI, uvicorn, pytest, httpx are not used.